sublime_jedi/helper.py

```diff
@@ -2,7 +2,22 @@
 import sublime
 import sublime_plugin
 
+import html
+
+try:
+    # mdpopups needs 3119+ for wrapper_class, which diff popup relies on
+    if int(sublime.version()) < 3119:
+        raise ImportError('Sublime Text 3119+ required.')
+    # mdpopups 1.9.0+ is required because of wrapper_class and templates
+    import mdpopups
+    if mdpopups.version() < (1, 9, 0):
+        raise ImportError('mdpopups 1.9.0+ required.')
+    _HAVE_MDPOPUPS = True
+except ImportError:
```

Comment:
Not sure whether this is required. If installed via Package Control, mdpopups will be installed and updated as well automatically. The import will most likely fail for people who installed manually.